docs/source/script_tutorial.rst

```diff
@@ -263,15 +278,17 @@ Graphical Tools
         # put control inside Menu
         menu.addLayoutWidget(control)
 
-    The only problem here is that you lose the close button for `control`, to
-    over come that menu has the layout sub-menu that allow the user to close
+    The only problem here is that you lose the close button for `control`. To
+    overcome this deficiency, the menu has a layout sub-menu so the user can close
     components inside the main menu.
 
-    Ok, may be you don't want to put components inside menu, your problem is
-    you want to close all windows at once, and not having to close each one.
-    To get that we use that PyQt closes all children windows of a window when
-    this is closed, so a good police is to pass menu as parent for all other
-    components (components always accept a parent key), for instance
+    Ok, maybe you don't want to put components inside menu. Your problem is
+    that you would like to close all windows at once and not each individually.
+    For this we leverage the fact that PyQt closes all children instances (windows) 
+    of an existing (parent) window. A good policy is to pass menu as the parent 
+    for all other components (components always accept a parent key)
+
```

Comment:
this is not working in python 2.6, as Qt is not accepting the parent key. But passing it to components is still always safe, just the wanted behavior will not work.